proxy.go

```diff
@@ -457,7 +457,7 @@ func (rp *reverseProxy) serveFromCache(s *scope, srw *statResponseWriter, req *h
 
 			rp.completeTransaction(s, statusCode, userCache, key, q, "")
 
-			err = RespondWithData(srw, reader, contentMetadata, 0*time.Second, tmpFileRespWriter.StatusCode(), labels)
+			err = RespondWithData(srw, reader, contentMetadata, 0*time.Second, false, tmpFileRespWriter.StatusCode(), labels)
```

Comment:
I wonder if in this case the value of the X-Cache header should be something else like N/A because even if it's a cache miss, it's something chproxy is not supposed to cache.